### ISSUE TITLE
Not able to open doors from inside

### ISSUE THREAD

**Issue by etrondriver** [electrichasgoneaudi/etron-issues#90]
## Description

Some owners has experienced that it is not possible to open doors from inside

## Online reports
[Audi e-tron Norge  Januar 21. 2021](https://www.facebook.com/groups/361111001113524/permalink/832451633979456/)

**Comment by ikanell** [comment]
Yes, this happened to me multiple times. Eventually I had to either turn the ignition off or unlock the car with the key in order to get the door to open. Audi claimed this was impossible and that it was my fault. This was one of the many problems that eventually made me give my e-tron back to Audi as a "lemon".
